File: C_Programming/Competitive_Programming/CP_012.c
////////////////////////////////////////////////////////////////////
//
//  File Name    : Display_Factors_Of_Number
//
//  Description  : This program accepts a number from the user
//                 and displays all factors of the given number,
//                 excluding the number itself.
//
//  Example:
//      Input  : 12
//      Output : 1 2 3 4 6
//
//      Input  : 20
//      Output : 1 2 4 5 10
//
////////////////////////////////////////////////////////////////////


#include<stdio.h>

void DisplayFactor(int iNo)
{
    int i = 0;

    if(iNo <= 0)
    {
        iNo = -iNo;
    }

    for(i = 1; i < iNo ; i++)
    {
        if(iNo % i == 0)
        {
            printf("%d\t", i);
        }
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number : \n");
    scanf("%d", &iValue);

    DisplayFactor(iValue);

    return 0;
}

File: Java_Programming/Competitive_Programming/CP_012.java
//////////////////////////////////////////////////////////////////
//
//  File Name    : Display_Grade_Based_On_Marks
//
//  Description  : This program accepts marks from the user
//                 and displays the corresponding grade
//                 based on the given marks.
//
//  Example:
//      Input  : 82
//      Output : B
//
//////////////////////////////////////////////////////////////////

class Logic
{
    void displayGrade(int iMarks)
    {
        if(iMarks >= 91 && iMarks <= 100)
        {
            System.out.println("A");
        }
        else if(iMarks >= 71 && iMarks <= 90)
        {
            System.out.println("B");
        }
        else if(iMarks >= 51 && iMarks <= 60)
        {
            System.out.println("C");
        } 
        else
        {
            System.out.println("D");
        }
    }
}

class CP_012
{
    public static void main(String A[])
    {
        Logic lobj = new Logic();

        lobj.displayGrade(82);
    }
}